addons/core/addon/decorators/notify.js

```diff
@@ -67,9 +70,11 @@ export function notifyError(notification, options = { catch: false }) {
           ? intlService.t(candidateKey)
           : candidateKey;
 
+        const sticky = options.sticky === undefined ? true : options.sticky;
```

Comment:
```suggestion
        const sticky = options.sticky ?? true;
```

Should we also add a default to the parameter since we already do it for catch as well? i.e. `options = { catch: false, sticky: true }`

Reply:
I like the idea of adding `sticky` to the default parameter.

Reply:
I agree. It could be confusing to see the comment that says it defaults to true but not see that in the signature.

Reply:
Well I mean he is technically defaulting to `true` currently, I think it needs to be done since someone can still always send in an object with just the `catch` and it won't default the sticky to true so we need to check it.

Reply:
Added additional explanation and signature.